react-native-in-app-payments-quickstart/README.md

```diff
@@ -56,6 +56,17 @@ Change to the quick start sample folder and install the plugin using `yarn`:
 
 Run the React Native project from the `react-native-in-app-payments-quickstart` project folder:
 
+  1. First Add an script in build phase:
```

Comment:
Change to "Add build phase to setup SquareInAppPaymentsSDK:"

Reply:
Done !